src/modules/uORB/uORBManager.cpp

```diff
@@ -368,14 +337,6 @@ int uORB::Manager::node_open(const struct orb_metadata *meta, const void *data,
 		return PX4_ERROR;
 	}
 
-	/*
-	 * Advertiser must publish an initial value.
-	 */
-	if (advertiser && (data == nullptr)) {
```

Comment:
I support the removal of this requirement, but we need to make sure the null-case is handled gracefully (in `uORB::Manager::orb_advertise_multi`)

Reply:
There should probably be one lower down in publish or even the DeviceNode write.

Reply:
Added data nullptr check in uORB::DeviceNode::publish.